flygym/envs/nmf_mujoco.py

```diff
@@ -26,13 +26,22 @@
 from flygym.terrain.mujoco_terrain import \
     FlatTerrain, Ball, GappedTerrain, ExtrudingBlocksTerrain
 from flygym.util.data import mujoco_groundwalking_model_path
-from flygym.util.data import default_pose_path, stretch_pose_path
-from flygym.util.config import all_leg_dofs
-
+from flygym.util.data import default_pose_path, stretch_pose_path, \
+    zero_pose_path
+from flygym.util.config import all_leg_dofs, all_tarsi_collisions_geoms, \
+    all_legs_collisions_geoms, all_legs_collisions_geoms_no_coxa
```

Comment:
`all_legs_collisions_geoms` and `all_legs_collisions_geoms_no_coxa` are not defined in `util/config.py`, unit test not passing.